llama_stack/cli/stack/remove.py

```
@@ -0,0 +1,118 @@
+        return distributions
+
+     def _list_stacks(self) -> None:
+        """Display available stacks in a table"""
+        distributions = self._get_distribution_dirs()
+        if not distributions:
+            print("No stacks found in ~/.llama/distributions")
+            return
+
+        headers = ["Stack Name", "Path"]
+        rows = [[name, str(path)] for name, path in distributions.items()]
+        print_table(rows, headers, separate_rows=True)
+
+     def _remove_stack_build_command(self, args: argparse.Namespace) -> None:
+        distributions = self._get_distribution_dirs()
+        
+        if args.all:
+            if not args.force:
+                confirm = input("Are you sure you want to delete ALL stacks? [y/N] ").lower()
+                if confirm != 'y':
+                    print("Deletion cancelled.")
+                    return
+            
+            for name, path in distributions.items():
+                try:
+                    shutil.rmtree(path)
+                    print(f"Deleted stack: {name}")
+                except Exception as e:
+                    print(f"Failed to delete stack {name}: {e}")
+            return
+
+        if args.list or not args.name:
+            self._list_stacks()
+            if not args.name:
+                return
+
+        if args.name not in distributions:
+            self._list_stacks()
+            print(f"Stack not found: {args.name}")
+            return
+
+        stack_path = distributions[args.name]
+        
+        if not args.force:
+            confirm = input(f"Are you sure you want to delete stack '{args.name}'? [y/N] ").lower()
+            if confirm != 'y':
+                print("Deletion cancelled.")
+                return
+
+        try:
+            shutil.rmtree(stack_path)
+            print(f"Successfully deleted stack: {args.name}")
+        except Exception as e:
+            print(f"Failed to delete stack {args.name}: {e}")
```

We should probably propagate the error and not return 0